easybuild/easyconfigs/b/Boost/Boost-1.72.0-gompi-2020a-rc2.eb

```diff
@@ -0,0 +1,25 @@
+name = 'Boost'
+version = '1.72.0'
+
+homepage = 'https://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'gompi', 'version': '2020a-rc2'}
```

Comment:
this should now be `2020a-rc3`, since https://github.com/easybuilders/easybuild-easyconfigs/pull/9985 was merged?

Reply:
yep, fixed in 991b05d